test/subscriptions/subscriptions-donations.js

```diff
@@ -1,91 +1,48 @@
 const { assertBn } = require('../helpers/asserts/assertBn')
 const { bn, bigExp } = require('../helpers/lib/numbers')
 const { buildHelper } = require('../helpers/wrappers/court')(web3, artifacts)
-const { assertRevert } = require('../helpers/asserts/assertThrow')
-const { assertAmountOfEvents } = require('../helpers/asserts/assertEvent')
-const { SUBSCRIPTIONS_ERRORS } = require('../helpers/utils/errors')
-const { SUBSCRIPTIONS_EVENTS } = require('../helpers/utils/events')
 
 const CourtSubscriptions = artifacts.require('CourtSubscriptions')
 const ERC20 = artifacts.require('ERC20Mock')
 
+// TODO: Delete these tests?
```

Comment:
I think we could delete them yes, Looks like we are just testing ERC20 token transfer now.